lua/core/utils/init.lua

```diff
@@ -49,8 +49,6 @@ local function load_module_file(module)
   return found_module
 end
 
---- user settings from the base `user/init.lua` file
-astronvim.user_settings = load_module_file "user.init"
```

Comment:
unsure how to keep this

Reply:
note: I searched all of github and did not see any non-fork using this variable

Reply:
Hm, yeah keeping this would definitely be more ideal so we can avoid tagging v3 of AstroNvim so soon